packages/core/contracts/Escrow.sol

```diff
@@ -47,6 +47,9 @@ contract Escrow is IEscrow, ReentrancyGuard {
     string public manifestUrl;
     string public manifestHash;
 
+    string public intermediateResultsUrl;
+    string public intermediateResultsHash;
```

Comment:
I wonder if hash of the files is actually necessary, what do you think?

Reply:
Agreed. Dropped `intermediateResultsHash` param. However, want to keep both function params of `storeResults` for backwards compatibility.